packages/server/src/sdk/app/rows/tests/internal.spec.ts

```diff
@@ -189,7 +190,6 @@ describe("sdk >> rows >> internal", () => {
             type: FieldType.AUTO,
             subtype: AutoFieldSubType.AUTO_ID,
             autocolumn: true,
-            lastID: 0,
```

Comment:
Decided to remove this because the code handles it being `undefined`. May as well exercise that.